src/pages/administration/datasources/index.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div>
+		<base-breadcrumb :inputs="breadcrumb" />
+		<h3>{{ $t("pages.administration.datasources.index.title") }}</h3>
+
+		<empty-state :image="imgsrc">
+			<template v-slot:description>
+				{{ $t("pages.administration.datasources.index.empty") }}
+			</template>
+		</empty-state>
+
+		<floating-fab
+			:position="position"
```

Comment:
https://storybook.test.schul-cloud.org/?path=/story/molecules-floatingfab--default

position should receive a bottom-right (default)

```suggestion
			:position="bottom-right"
```